packages/composable-controller/CHANGELOG.md

```diff
@@ -25,10 +25,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - **BREAKING:** Passing a non-controller into `controllers` constructor option now throws an error ([#3904](https://github.com/MetaMask/core/pull/3904))
-- **BREAKING:** The `AllowedAction` parameter of the `ComposableControllerMessenger` type is narrowed from `string` to `never`, as `ComposableController` does not use any external controller actions ([#3904](https://github.com/MetaMask/core/pull/3904))
+- **BREAKING:** The `AllowedActions` parameter of the `ComposableControllerMessenger` type is narrowed from `string` to `never`, as `ComposableController` does not use any external controller actions. ([#3904](https://github.com/MetaMask/core/pull/3904))
```

Comment:
It seems that we are updating the changelog for an older version, is this intentional? Are there changes we forgot to include for this version?

Reply:
Fixed to align with final changelog in PR description: https://github.com/MetaMask/core/pull/3952/commits/365aa0214cce99a9065e2901738ac3116b2619a5